### ISSUE TITLE
Add details to `data` documentation

### ISSUE THREAD

**Issue by geanders** [rickzhouwen/HDtest#4]
For the two datasets, `GO26` and `GO54`, it would be very helpful for us to add more documentation. Is there somewhere I could look to find out more about these datasets to work on that? For example, do you use them as examples in one of your papers?

**Comment by rickzhouwen** [comment]
I fully agree with you. They are the GO terms used as examples in a paper (like mine in biometrics: http://onlinelibrary.wiley.com/doi/10.1111/biom.12552/abstract). Their information are here: http://amigo.geneontology.org/amigo/term/GO:0034080; http://amigo.geneontology.org/amigo/term/GO:0016032.

I will add some more details to them; also the description parts as we discussed last week. I really need to learn how to do it in R-studio and push same time, I have tried a few times but did not make it...
